components/nexus-ehcache/src/main/java/org/sonatype/sisu/ehcache/CacheManagerComponentImpl.java

```diff
@@ -48,8 +50,8 @@
   private CacheManager cacheManager;
 
   @Inject
-  public CacheManagerComponentImpl() throws IOException {
-    this(null);
+  public CacheManagerComponentImpl(final ApplicationDirectories directories) throws IOException {
+    this(directories.getAppDirectory("etc/ehcache.xml"));
```

Comment:
presumably this should also be changed: https://github.com/sonatype/nexus-oss/blob/master/components/nexus-elasticsearch/src/main/java/org/sonatype/nexus/elasticsearch/internal/NodeProvider.java#L79 ?